<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nowPlayingLabel">
         <property name="text">
          <string>Now Playing: Sicko Mode by Kanye West</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QPushButton" name="lightMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="taskList">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::DropAction::MoveAction</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="shuffleButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previousSongButton">
           <property name="text">
            <string> Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pausePlayButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextSongButton">
           <property name="text">
            <string> Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="volumeLabel">
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="volumeSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="taskNameLabel">
      <property name="text">
       <string>Task Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskInput"/>
    </item>
    <item>
     <widget class="QLabel" name="dueDateLabel">
      <property name="text">
       <string>Due Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dueDate"/>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resources/icons/Add.png</normaloff>:/resources/icons/Add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTo_Do_List">
    <property name="title">
     <string>To-Do List</string>
    </property>
   </widget>
   <addaction name="menuTo_Do_List"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_addButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addButton_clicked()</slot>
  <slot>on_deleteButton_clicked()</slot>
 </slots>
</ui>
